packages/fxa-settings/src/lib/auth-errors/auth-errors.test.ts

```diff
@@ -0,0 +1,52 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { composeAuthUiErrorTranslationID, AuthUiErrorNos } from './auth-errors';
+
+
+ describe('composeAuthUIErrorTranslationID', () => {
+   it('returns an empty string if passed no error number', () => {
+    let errorWithNoErrorNumber = { message: "I'm an error with no error number!" };
+    expect(
+      composeAuthUiErrorTranslationID(errorWithNoErrorNumber)
+    ).toEqual('');
+   });
+
+   it('returns an empty string when given an error number with no corresponding error in the AuthUiError object', () => {
+    const notAnExistingErrorNumber = 100000;
+    const errorWithAnInvalidErrorNumber = { errno: notAnExistingErrorNumber, message: 'This is an error with an invalid errno!'};
+    // First we just check that this has not become a valid AuthUiError number, for the sake of future clarity.
+    expect(AuthUiErrorNos[notAnExistingErrorNumber]).toBeUndefined();
+    // Then we actually test the behavior.
+    expect(
+      composeAuthUiErrorTranslationID(errorWithAnInvalidErrorNumber)
+    ).toEqual('');
+   });
+
+   it('correctly forms and returns a translation id for an auth ui error string which has never been updated (has no version number)', () => {
+    const validErrorNo = 106;
+    // First we just make sure that this is still a valid UI error.
+    expect(AuthUiErrorNos[validErrorNo]).toBeDefined();
+    // Then we make sure that it doesn't have a version number. This is a safeguard against the test unknowingly being outdated.
+    expect(AuthUiErrorNos[validErrorNo].version).toBeUndefined();
+    // The we actually test the functionality.
+    const expectedStringId = 'auth-error-106';
+    const stringId = composeAuthUiErrorTranslationID({ message: 'I am a valid error with a valid error number', errno: validErrorNo });
+    expect(stringId).toEqual(expectedStringId);
+   });
+
+   it('correctly forms and returns a translation id for an auth ui error string which HAS been updated (has a version number', () => {
+    const errnoWithVersion= 105;
```

Comment:
Do you have Prettier auto-formatting stuff for you on save? There's at least this one weird spot showing `errnoWithVersion= 105`.